src/components/App/App.jsx

```diff
@@ -1,23 +1,21 @@
-import React from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import Portfolio from '../Portfolio/Portfolio.jsx';
-import Item from '../Item/Item.jsx';
-import Outliner from '../Outliner/Outliner.jsx';
+import React from 'react'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import Portfolio from '../Portfolio/Portfolio.jsx'
+import Item from '../Item/Item.jsx'
+import Outliner from '../Outliner/Outliner.jsx'
```

Comment:
Don't edit lines if it is not part of the feature you are implementing (it would mess completely the "blame" feature, and moreover it makes edition conflicts more likely).